docs/HowTo/BreakingChanges.md

```diff
@@ -0,0 +1,142 @@
+# Breaking Changes
+
+Compatibility is a very important goal of each .NET release.  In order to achieve this, various techniques are employed. As shown
+earlier, there is additive functionality with each version, so previous versions will still work.  However, when a change must be 
+made to previous functionality (for performance, security, or a bug) this can cause compatibility problems in already written code. 
+A distinction to understand with .NET compatibility is runtime vs retargeting issues.
+
+When developing an app, the .NET Framework version that the app is targeting is [specified either by specifying a target Framework 
+in Visual Studio](https://msdn.microsoft.com/en-us/library/bb398202%28v=vs.140%29.aspx), directly 
+[specifying the target Framework](https://msdn.microsoft.com/en-us/library/hh264221.aspx) in a project file, or applying a 
+[TargetFrameworkAttribute](https://msdn.microsoft.com/en-us/library/system.runtime.versioning.targetframeworkattribute%28v=vs.110%29.aspx) 
+to the source code.
+
+An app that is built targeting a particular version of the .NET Framework (4.0 or above) will run on that version of the .NET 

```

Comment:
You may want to drop the sentence beginning 'An app that is built targeting a particular...'. That statement is usually true, but depends on 'supported runtime' attributes in the app.config, and can be changed separate from things like TFM.